blog/docs/events/virtual-office-hour-12-2022.md

```diff
@@ -0,0 +1,46 @@
+# First 🌱 Knative Virtual Office Hour
+
+**Date: 2022-12-01**
+
+**Authors: [Alek Slominski](https://aslom.net/), [Evan Anderson](https://off-by-one.dev/), [Lance Ball](https://twitter.com/lanceball), and Paul Schweigert**
+
+We think that there should be an easy way for new Knative users and contributors to get started with our project. Holding virtual office hours for questions and help with using or contributing to Knative should help to get started with using code and contributing.
+
+During the recent KubeCon NA 2022 in Detroit we had Knative Day, Maintainer Meetup, and ContribFest that showed there are people interested in using or contributing to Knative. However, today we do not have any ongoing online meetings for people to just show up. They can join Slack but that is asynchronous and requires joining yet another Slack, then reading channels, and figuring where to ask your questions. Driving your own engagement can be a barrier to participating in the community.
+
+Therefore, we are organizing the first Knative Virtual Office Hour in 2022.
+
+**When:** [December 9 (Friday) 11:30 AM US Eastern](https://dateful.com/convert/utc?t=430pm&d=2022-12-09)
+**How to join:** Zoom link will be posted in Slack and knative-user mailing list
```

Comment:
We should go back and update this when we have the calendar, but there's no reason not to post the blog post now.

Reply:
:+1: